lessons/Algebra2/exploring-linear-relationships/langs/en-us/index.adoc

```diff
@@ -0,0 +1,239 @@
+= Exploring Linear Relationships
+
+@description{Students investigate linear relationships in demographic data about US States, using an inquiry-based model of sense-making, hypothesizing, modeling and testing.}
+
+@lesson-prereqs{scatter-plots}
+
+@keywords{}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Read and interpret real-world data, presented in a scatter plot
+* Describe correlations as Strong, Moderate, or Weak
+* Model linear relationships using linear functions
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Pyret to make predictions with linear models
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* All students should log into @starter-file{program-list} and, open the @online-exercise{https://code.pyret.org/editor#share=10bWYM4rOq_ZshtX6_M-xf-bPbfz61rd1, State Demographics Starter File}.
+
+| Key Points For The Facilitator
+|
+* TBD
+|===
+
+== Looking for Patterns @duration{15 minutes}
+
+=== Overview
+Students explore the State Demographics dataset, and create scatter plots to search for relationships between columns. They share and discuss their findings with the class.
+
+=== Launch
+
+[.lesson-instruction]
+- Today we'll be looking at data from each state in the US, and searching for possible relationships within the data!
+- Open the @online-exercise{https://code.pyret.org/editor#share=10bWYM4rOq_ZshtX6_M-xf-bPbfz61rd1, State Demographics Starter File}
+- From the File menu, select "Save a Copy", and click "Run".
+- Working in pairs or small groups, complete questions 1 and 2 in @printable-exercise{exploring-states.adoc}.
+
+
+=== Investigate
+
+Have students share back what they Notice and Wonder about the dataset.
+
+The code on lines 13 and 14 show us how Pyret defines values:
+
+```
+alabama = row-n(states-table, 0)
+alaska  = row-n(states-table, 1)
+```
+
+Just as in math, the code `x = 4` will define a variable `x` to be the value `4`. But in Pyret, valus can be more than just numbers! The variables `alabama` and `alaska` were _rows_ from the table.
+
+[.lesson-instruction]
+- Working in pairs or small groups, complete question 3 (making displays and suggesting possible relationships) in @printable-exercise{exploring-states.adoc}.
+- **Do not go on to question 4!** We'll be returning to that later in the lesson.
+
```

Comment:
Students are _supposed_ to make scatter plots here. Where do the directions say otherwise?

I like the idea of writing those questions explicitly in the workbook page, rather than calling them out in the lesson. Done.

Reply:
The workbook page says "STOP!" before they get to the scatter plots. Then, they don't return to this page until much later in the lesson.